playbooks/demo_copy_and_install_vios.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: HMC create and activate logical partition
+  hosts: hmcs
+  connection: local
+  vars:
+      curr_hmc_auth:
+          username: hscroot
+          password: !vault |
+              $ANSIBLE_VAULT;1.1;AES256
+              34656335616165343966323739633761653065346138623233633764353636373063313736373563
+              6130373162376464666634363734303164396364656535350a663661636663323434386466383634
+              63323432303833643864396130393133346666336661346432356433353035663038643833373332
+              3664333762656138640a306539363765343261383363346532643639383938656434333564323139
+              6365
+
+  tasks:
+      - name: Copy vios installation inamge from sftp location to HMC disk
+        ibm.power_hmc.vios:
+            hmc_host: '{{ inventory_hostname }}'
+            hmc_auth: "{{ curr_hmc_auth }}"
+            media: sftp
+            directory_name: "vios_3.1.0.10"
+            remote_server: 9.3.115.83
+            sftp_auth:
+              username: sftp_username
+              password: sftp_password
+            remote_directory: /home/hmcct/
+            files:
+               - flash.iso
+            action: copy
+        register: testout
+
+      - name: Stdout the result of copy
+        ansible.builtin.debug:
+            msg: '{{ testout }}'
+
+      - name: Install the VIOS using image available in disk
```

Comment:
Can you rewrite the playbook name as with more details like which version of vios is being installed